sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -28,6 +28,25 @@
            'OrderedPositionRangeTracker', 'UnsplittableRangeTracker']
 
 
+class OffsetRange(object):
+
+  def __init__(self, start, stop):
+    self.start = start
+    self.stop = stop
+
+  def split(self, desired_num_offsets_per_split, min_num_offsets_per_split):
+    current_split_start = self.start
+    max_split_size = max(desired_num_offsets_per_split,
```

Comment:
What's the difference between "offsets per split" and "size?" Maybe just use consistent terminology?